chcemvediet/apps/inforequests/tests/test_models/test_inforequest.py

```diff
@@ -734,6 +758,28 @@ def test_newest_undecided_email_property_with_no_undecided_emails(self):
         with self.assertNumQueries(0):
             self.assertIsNone(inforequest.newest_undecided_email)
 
+    def test_last_action_property(self):
+        inforequest, _, actions = self._create_inforequest_scenario(
+                (u'request', dict(created=naive_date(u'2012-10-05'))),
+                (u'refusal', dict(created=naive_date(u'2012-10-05'))),
+                (u'appeal', dict(created=naive_date(u'2012-10-07'))),
+                (u'remandment', dict(created=naive_date(u'2012-10-06'))),
+        )
+        _, _, appeal, _ = actions
+        self.assertEqual(inforequest.last_action, appeal)
+
+    def test_disclosure_level_property(self):
+        inforequest, _, _ = self._create_inforequest_scenario(
+            (u'disclosure', dict(disclosure_level=Action.DISCLOSURE_LEVELS.NONE)),
+            (u'disclosure', dict(disclosure_level=Action.DISCLOSURE_LEVELS.PARTIAL)),
+            (u'disclosure', dict(disclosure_level=Action.DISCLOSURE_LEVELS.FULL)),
```

Comment:
By sa zisiel aj test, ktory overi, ze sa berie disclosure level nie len z `disclosure` akcii. Ale zo vsetkych akcii. Pretoze napriklad aj taky `reversion` moze mat kludne disclosure level. Si pozri wizard, ktore vsetky typy akcii sa pytaju, ci boli nejake informacie zverejnene a otestuj, ze sa beru do uvahy.

A asi by sa zislo aj upravit komentar pri tej property. Namiesto "any disclosure action" napisat "any action that may disclose information"

Reply:
:ok: napisal som test, kde disclosure_level som priradil kazdemu typu akcie. Sice to nie je logicky (podla wizardu) spravne, ucel testu to splnilo. Je uz len na nas aby sme v projekte disclosure_level priradovali vybranym typom akcii. Takisto v test_action sa disclosure_level testuje na `Action.TYPES.REQUEST`